gcloud-java-datastore/src/test/java/com/google/gcloud/datastore/LocalDatastoreHelperTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud.datastore;
+
+import static org.junit.Assert.assertSame;
+import static org.junit.Assert.assertTrue;
+
+import com.google.gcloud.AuthCredentials;
+import com.google.gcloud.datastore.testing.LocalDatastoreHelper;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class LocalDatastoreHelperTest {
+
+  private static final double TOLERANCE = 0.00001;
+  private static final String PROJECT_ID_PREFIX = "test-project-";
```

Comment:
Just realized that this test class does not match the same package structure as the class it tests.
I think we should change that. See #880 
And when we do so, you can change `LocalDatastoreTestHelper.PROJECT_ID_PREFIX` to be package scope and use it here.